Nxo fix fallback (#1069)
braintree/braintree_ios: CHANGELOG.md

## unreleased
* Expose reference documentation for `BTAppContextSwitcher.handleOpen(_:)` and `BTAppContextSwitcher.handleOpenURL(context:)`

> * Fixed a bug where post vaulting FirstName, LastName and email are null but a nonce is generated.

Reviewer: ```suggestion
* Fixed a bug to return `firstName`, `lastName`, `email`, and `payerID` in `BTPayPalNativeCheckoutAccountNonce` as expected.
```

Reviewer 2: Or this! 👏

> * Fixed a bug where post vaulting FirstName, LastName and email are null but a nonce is generated.

Reviewer 2: ```suggestion
* Fix a bug where the following properties on `BTPayPalNativeCheckoutAccountNonce` were always returning nil after vaulting: `firstName`, `lastName`, and `email`
```

> * Fixed a bug to return `firstName`, `lastName`, `email`, and `payerID` in `BTPayPalNativeCheckoutAccountNonce` as expected.

Reviewer 2: ```suggestion
* Fixed a bug to return `firstName`, `lastName`, `email`, and `payerID` on `BTPayPalNativeCheckoutAccountNonce` when available
```

> * Fixed a bug to return `firstName`, `lastName`, `email`, and `payerID` in `BTPayPalNativeCheckoutAccountNonce` when available.

Reviewer 2: ```suggestion
* Fixed a bug to return `firstName`, `lastName`, `email`, and `payerID` on `BTPayPalNativeCheckoutAccountNonce` when available.
```

* BraintreePayPalNativeCheckout (General Availability release)
  * Update PayPalCheckout from 0.110.0 to 1.0.0. This is our newly released General Availability version
     * _Note: This module will now be subject to semantic versioning_

>   * Fixed a bug where post vaulting FirstName, LastName and email are null but a nonce is generated.

Reviewer: We will want to move this into a new `## unreleased` section since this section was released in [version 6.3.0](https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md#630-2023-07-10). Please ensure you've merged the latest from main into this branch to pull in the updated changelog from the version that was released.
